lib/charms/lte_core_interface/v0/lte_core_interface.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Library for the `lte-core` relation.
+
+This library contains the Requires and Provides classes for handling the `lte-core`
+interface.
+
+The motivation for creating this library comes from the need of relating charmed EPCs with
+charmed simulated enodeBs and user equipment (UEs), for testing purposes. The interface will share
+the IP address of the MME (Mobility Management Entity) from EPC the to the charm which contains the
```

Comment:
one extra 'the' :)
```suggestion
the IP address of the MME (Mobility Management Entity) from EPC to the charm which contains the
```

Reply:
Agree, however I've done minor tweaks. Let me know what you think.

Reply:
Fixed.